data_analysis/plots.py

```diff
@@ -0,0 +1,171 @@
+from pathlib import Path
+from typing import Union
+
+import matplotlib
+matplotlib.use('QtAgg')
+
+import logging
+import folium
+import geopandas as gpd
+import pandas as pd
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import compare_scheduled_and_rt
+import static_gtfs_analysis
+
+CHICAGO_COORDINATES = (41.85, -87.68)
+PLOTS_PATH = Path(__file__).parent.parent / 'plots' / 'scratch'
+
+logger = logging.getLogger()
+logging.basicConfig(level=logging.INFO)
+
+
+def n_routes(df: Union[pd.DataFrame, gpd.GeoDataFrame],
+             n: int = 10,
+             lowest: bool = True) -> Union[pd.DataFrame, gpd.GeoDataFrame]:
+    """Returns the n route_ids with the lowest or highest ratio
+       of completed trips
+
+    Args:
+        df (Union[pd.DataFrame, gpd.GeoDataFrame]): DataFrame with ratio
+            of completed trips by route_id
+        n (int, optional): number of route_ids to return . Defaults to 10.
+        lowest (bool, optional): whether to return the lowest ratios
+    Returns:
+        Union[pd.DataFrame, gpd.GeoDataFrame]: A DataFrame containing the
+            n worst routes.
```

Comment:
nitpick: I think this could be n worst or n best?

Reply:
What about something like `n_worst_best_routes`?